src/Marten/Internal/Sessions/AmbientTransactionLifetime.cs

```diff
@@ -26,20 +34,31 @@ public AmbientTransactionLifetime(SessionOptions options)
         {
             OwnsConnection = true;
         }
+
+        CommandTimeout = _options.Timeout ?? _connection?.CommandTimeout ?? 30;
```

Comment:
**SUGGESTION:** It might be worth putting `30`  value as some general const.

Reply:
Yes, and I also think we should move that default way down to maybe 5. 30 seems crazy excessive, but it's what we had before.